aiven/resource_service_user.go

```diff
@@ -120,11 +131,39 @@ func resourceServiceUserCreate(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
+	if newPassword, ok := d.GetOk("password"); ok {
+		_, err := client.ServiceUsers.Update(projectName, serviceName, username,
```

Comment:
I was going to say this should just be passed in the create call but indeed create does not allow setting password, which is quite daft.